std/node/util/main_test.ts

```diff
@@ -0,0 +1,122 @@
+import { test } from "../../testing/mod.ts";
+import { assert } from "../../testing/asserts.ts";
+import * as util from "./main.ts";
```

Comment:
main.ts ? I don’t think we should use the file name main.ts in std/node ...

Reply:
Yeah, direct port, i'll change this as mentionned in the other comment